server/service/src/sync/test/test_data/invoice_line.rs

```diff
@@ -99,24 +99,26 @@ fn trans_line_1_pull_record() -> TestSyncPullRecord {
 fn trans_line_1_push_record() -> TestSyncPushRecord {
     TestSyncPushRecord {
         change_log: ChangelogRow {
-            id: 2,
+            cursor: 2,
             table_name: ChangelogTableName::InvoiceLine,
-            row_id: TRANS_LINE_1.0.to_string(),
+            record_id: TRANS_LINE_1.0.to_string(),
             row_action: ChangelogAction::Upsert,
+            name_id: Some(mock_outbound_shipment_a().name_id),
+            store_id: Some(mock_outbound_shipment_a().store_id),
         },
         push_data: json!(LegacyTransLineRow {
-            ID: TRANS_LINE_1.0.to_string(),
-            transaction_ID: "outbound_shipment_a".to_string(),
-            item_ID: mock_item_a().id,
+            id: TRANS_LINE_1.0.to_string(),
+            invoice_id: "outbound_shipment_a".to_string(),
```

Comment:
Are `transaction_id` and `invoice_id` the same? I would have thought they'd be on primary keys on different tables?

Reply:
it's transaction_id in incoming sync record (and `transact` table in mSupply is invoice table in omSupply), we really wanted to avoid confusion with database transactions and an entity we call transaction, so name the new table invoice.

Anyways, in the changes there I've used serde for renaming, vs doing manual renaming twice (in pull and push)